src/core/server/opensearch/client/retry_call_cluster.test.ts

```diff
@@ -62,13 +62,13 @@ describe('retryCallCluster', () => {
       ...dummyBody,
     });
 
-    client.asyncSearch.get
```

Comment:
NIT: It's just a test file but it seems like we removed asyncSearch but these tests are made for asyncSearch. Do we need them anymore?

Reply:
Yeah I'd say if we're testing `search` elsewhere these can be removed.